mkosi/__init__.py

```diff
@@ -7546,9 +7546,21 @@ def find_ovmf_vars() -> Path:
     die("Couldn't find OVMF UEFI variables file.")
 
 
+def qemu_check_kvm_support() -> bool:
+    kvm = Path("/dev/kvm")
+    if not kvm.exists():
```

Comment:
Not sure, but I guess `/dev/kvm` is always a character device, if so we could make this more precise. 
```suggestion
    if not kvm.is_char_device():
```
I don't know whether this would already obviate the attempted solution below

Reply:
Probably a good idea to add it but I don't think it hurts to still do the below check.